SECURITY_RELEASE_PROCESS.md

```diff
@@ -188,20 +188,28 @@ could be in the form of the following:
 To be eligible for the cncf-envoy-distributors-announce mailing list, your
 distribution should:
 
-1. Be an actively maintained distribution of Envoy components OR offer Envoy as a publicly
-   available service in which the product clearly states that it is built on top of Envoy. E.g.,
-   "SuperAwesomeLinuxDistro" which offers Envoy pre-built packages OR
-   "SuperAwesomeCloudProvider's Envoy as a Service (EaaS)". A cloud service that uses Envoy for a
-   product but does not publicly say they are using Envoy does not qualify.
+1. Be either:
+   1. An actively maintained distribution of Envoy components. An example is
+   "SuperAwesomeLinuxDistro" which offers Envoy pre-built packages. Another example is
+   "SuperAwesomeServiceMesh" which offers a service mesh product that includes Envoy as a component.
+   2. OR offer Envoy as a publicly available infrastructure or platform service, in which the
```

Comment:
Nit: you might want to put `OR` in a separate line above `2.` (assuming it renders correctly).